rmk/src/config/mod.rs

```diff
@@ -82,15 +84,54 @@ pub struct TapHoldConfig {
     pub prior_idle_time: Duration,
     pub post_wait_time: Duration,
     pub hold_timeout: Duration,
+    pub permissive_hold: bool,
+    pub chordal_hold: bool,
+}
+
+#[derive(Debug, Copy, Clone, PartialEq, Eq, PartialOrd, Ord)]
+#[cfg_attr(feature = "defmt", derive(defmt::Format))]
+pub enum ChordHoldHand {
+    Left,
+    Right,
+}
+
+#[derive(Clone, Debug)]
+pub struct ChordHoldState<const COLS: usize> {
+    pub hand: ChordHoldHand,
+}
+
+impl<const COLS: usize> ChordHoldState<COLS> {
```

Comment:
Sometimes the keyboard are split by rows, not cols, for example: https://github.com/eduardobraun/corners/blob/main/keyboard.toml